borg/testsuite/helpers.py

```diff
@@ -665,15 +666,45 @@ def dotest(test_archives, n, skip, indices):
 
 
 class PruneWithinTestCase(BaseTestCase):
+    def test_interval(self):
```

Comment:
as interval() is a general purpose validator now, maybe move the tests related to it to a separate top-level test outside of this test class.